R/read_AHCCD_daily_temp.R

```diff
@@ -0,0 +1,161 @@
+#' Reads AHCCD daily temperature file
+#'
+#' @description This program reads Adjusted and Homogenized Canadian Climate Data (AHCCD) of daily 
+#' min, max or mean air temperatures. The values are arranged as
+#' month x day, which makes them difficult to read using standard \R functions.
+#' @param temp_file Required. Name of the file to be read.
+#' @return If successful, returns the values in a dataframe, consisting of the date, 
+#' the value and the data code. If unsuccessful, returns the value \code{FALSE}.
+#' @author Kevin Shook
+#' @seealso  \code{\link{read_AHCCD_daily_temp}} \code{\link{read_AHCCD_monthly_precip}} 
+#' @references Monthly AHCCD data are available from \url{http://www.ec.gc.ca/dccha-ahccd}. 
+#' Daily values must be requested. Any use of the temperature data must cite 
+#' \cite{Vincent, L. A., X. L. Wang, E. J. Milewska, H. Wan, F. Yang, and V. Swail, 2012. 
+#' A second generation of homogenized Canadian monthly surface air temperature for climate 
+#' trend analysis, J. Geophys. Res., 117, D18110, doi:10.1029/2012JD017859.}
+#' @examples
+#' \dontrun{
+#'stoon_tmax <- read_AHCCD_daily_temp("dx40657120.txt")}
+#' @export
+
+read_AHCCD_daily_temp <- function(temp_file){
```

Comment:
You might also consider writing some helper functions as there is some redundant code between each function.

Reply:
Also a good idea.

Reply:
Done!